metrics/ttfp/track_pr_ttfp.jl

```diff
@@ -0,0 +1,19 @@
+using Pkg
+cd(@__DIR__)
+Pkg.activate(".")
+pkg"dev ../../MakieCore/ ../../ ../../CairoMakie/; add GeometryBasics#sd/no-sarray"
+
+using Pkg
+Pkg.activate("./makie-tagged")
+pkg"add CairoMakie"
+Pkg.precompile("CairoMakie")
+
+result_1_7 = read(`julia --project=. ./benchmark-ttfp.jl`, String)
+result_th = read(`./../../../../julia_th/julia --project=. ./benchmark-ttfp.jl`, String)
```

Comment:
I'm guessing most people won't have this directory on their system. (Glad you're testing it, though!)